src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBlob.java

```diff
@@ -44,8 +44,10 @@
 
     static private final Logger logger = Logger.getLogger("com.microsoft.sqlserver.jdbc.internals.SQLServerBlob");
 
-    static private final AtomicInteger baseID = new AtomicInteger(0);	// Unique id generator for each instance (used for logging).
+    static private final AtomicInteger baseID = new AtomicInteger(0);   // Unique id generator for each instance (used for logging).
     final private String traceID;
+    
+    private InputStream outputStream = null;
```

Comment:
rename this variable, using `outputStream` as variable name for `InputStream` is confusing.